test/tests/generic/test_crud_client.go

```diff
@@ -30,32 +32,37 @@ func TestCrudClient(namespace1, namespace2 string, client ResourceClient, opts c
 	input.Metadata.Labels = labels
 
 	err := client.Register()
-	Expect(err).NotTo(HaveOccurred())
+	ExpectWithOffset(testOffset, err).NotTo(HaveOccurred())
+
+	// list with no resources should return empty list, not err
+	list, err := client.List(namespace1, clients.ListOpts{})
```

Comment:
This is the functional change of this PR. Before we listed resources in the "" namespace. Now we make subsequent calls to the specific namespace.